src/actions/users.js

```diff
@@ -224,6 +224,8 @@ export function getProfiles(offset, limit = General.PROFILE_CHUNK_SIZE) {
             return null;
         }
 
+        const profiles = profileListToMap(profilesList);
```

Comment:
Should we maybe be changing the reducers to handle a list or profiles instead of converting the list into a map and then passing it down?